src/lib/Libifl/pbsD_connect.c

```diff
@@ -592,7 +667,46 @@ __pbs_disconnect(int connect)
 	if (pbs_client_thread_destroy_connect_context(connect) != 0)
 		return -1;
 
-	(void)destroy_connection(connect);
+	destroy_connection(connect);
+
+	return 0;
+}
+
+/**
+ * @brief
+ *	-send close connection batch request
+ *
+ * @param[in] connect - socket descriptor
+ *
+ * @return	int
+ * @retval	0	success
+ * @retval	-1	error
+ *
+ */
+int
+__pbs_disconnect(int connect)
+{
+	svr_conn_t *svr_conns = NULL;
+
+	if (connect <= 0)
```

Comment:
isn't 0 valid fd?

Reply:
0, 1 and 2 are reserved (so ideally it should be <=2), but here i can just check for -1 i think.